src/main.rs

```diff
@@ -96,6 +104,18 @@ async fn main() -> Result<()> {
             let review = prr.get_pr(&owner, &repo, pr_num, force).await?;
             println!("{}", review.path().display());
         }
+        Command::Edit { pr, force } => {
+            let (owner, repo, pr_num) = prr.parse_pr_str(&pr)?;
+            let review = prr.get_pr(&owner, &repo, pr_num, force).await?;
+            let editor = env::var("EDITOR")?;
+            let path = review.path();
+            std::process::Command::new(editor)
```

Comment:
Probably want to move this code into `prr.rs` if we wnat to share open functionality between `prr-open` and `prr get --open`